### ISSUE TITLE
Quarto documents that include library(qualtRics) crash on deployment to shinyapps

### ISSUE THREAD

**Issue by ELJRussell** [ropensci/qualtRics#357]
Not much reprex needed here - I had a document that compiled fine on my local server, and then once I deployed to shinyapps, it would not run. I ended up deleting all qualtRics code, and found that it would deploy when I did not have library(qualtRics), but would not deploy when I added that line.

**Comment by juliasilge** [comment]
I don't believe that static Quarto documents are supported on shinyapps.io at all. Check out:

- this community post: https://forum.posit.co/t/issues-deploying-quarto-documents/143444/3
- supported options for publishing: https://quarto.org/docs/publishing/

Can you share a little more about what you are experiencing? Can you create a [reprex](https://reprex.tidyverse.org/) (a minimal reproducible example) for this? The goal of a reprex is to make it easier for us to recreate your problem so that we can understand it and/or fix it. If you've never heard of a reprex before, you may want to start with the [tidyverse.org help](https://www.tidyverse.org/help/) page. Thanks! 🙌

**Comment by juliasilge** [comment]
Let us know if you have further questions! 🙌
